### PR TITLE
Removes Greytide

### PR DESCRIPTION
**Pull request by Expletive**
:cl:
balance: You can no longer join as an assistant.
balance: There can now be an unlimited number of clowns.
/:cl:

Assistants are terrible. Nobody likes them, and they generally do nothing but ruin the game for everyone else. Therefore, I propose replacing them all with clowns.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
I would actually prefer this in reverse, clowns are untrustworthy.

File: code/modules/jobs/job_types/civilian.dm [hunk 1]
```diff
@@ -128,7 +128,7 @@ Curator
 	outfit = /datum/outfit/job/curator
 
 	access = list(GLOB.access_library)
-	minimal_access = list(GLOB.access_library)
+	minimal_access = list(GLOB.access_library, GLOB.access_construction,GLOB.access_mining_station)
```
Comment (Jalleo) [inline code/modules/jobs/job_types/civilian.dm]: uhhh

**Comment by Expletive** [comment]
Oh, I've fucked up. Those first three commits shouldn't be in there. I'm really no good at this github thing.

**Comment by Expletive** [comment]
I apparently fucked up, and the commits from the first PR I made are also attached to this one.

**Comment by FantasticFwoosh** [comment]
Take it slower next time, separate out your branches (where your related PR's are held) before you pull PR & read our contributing guidelines please.

**Pull request closed without merging**